packages/astro/src/webcontainer-files.ts

```diff
@@ -63,12 +64,12 @@ export class WebContainerFiles {
 
     await Promise.all(
       folders.map(async (folder) => {
-        const fileRef = getFilesRef(folder, { contentDir, templatesDir });
```

Comment:
I tried to normalize the name so it's now `filesRef` to refer to the string that is conceptually a "reference" to a set of files.

After doing that I also realised that the type `FilesRef` is incorrect and should probably be `FilesRefWithFileList` but I don't like that one so I haven't renamed it just yet.

cc @SamVerschueren

Reply:
I can't come up with something better than `FilesRefWithFileList`. So I would go for that name (for now). We could also just name it `FilesRefList` or `FilesRefMap`.